sentry-android-ndk/src/main/java/io/sentry/android/ndk/SentryNdk.java

```diff
@@ -19,9 +21,18 @@ private SentryNdk() {}
     System.loadLibrary("sentry-android");
   }
 
-  private static native void initSentryNative(SentryOptions options);
+  private static native void initSentryNative(@NotNull final SentryOptions options);
 
-  public static void init(SentryOptions options) {
+  public static void init(@NotNull final SentryOptions options) {
+    addPackage(options);
     initSentryNative(options);
   }
+
+  private static void addPackage(@NotNull final SentryOptions options) {
+    final SdkVersion sdkVersion = options.getSdkVersion();
+    if (sdkVersion == null) {
+      return;
+    }
+    sdkVersion.addPackage("maven:sentry-android-ndk", BuildConfig.VERSION_NAME);
+  }
```

Comment:
`l`: We could only pass in the `sdkVersion`. `addPackage` doesn't need to know about `SentryOptions`. 
```suggestion
  private static void addPackage(final SdkVersion sdkVersion) {
    if (sdkVersion == null) {
      return;
    }
    sdkVersion.addPackage("maven:sentry-android-ndk", BuildConfig.VERSION_NAME);
  }
```

Reply:
makes sense